Common/Util/ResolutionTimeSpan.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+
+namespace QuantConnect.Util
+{
+    /// <summary>
+    /// This class allows implicit conversions between Resolution and TimeSpan
+    /// </summary>
+    public class ResolutionTimeSpan
+    {
+        public ResolutionTimeSpan(TimeSpan timeSpan) { val = timeSpan; }
+        public TimeSpan val;
```

Comment:
Since this should probably be a struct, let's make this `private readonly TimeSpan _value;` -- following along with the project's code style.